tests/utils/IntPointTest.h

```diff
@@ -0,0 +1,50 @@
+//Copyright (c) 2017 Tim Kuiper
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#ifndef INT_POINT_TEST_H
+#define LINEARALG2DTEST_H
+
+#include <cppunit/TestFixture.h>
+#include <cppunit/extensions/HelperMacros.h>
+#include <../src/utils/intpoint.h>
+
+namespace cura
+{
+
+class IntPointTest : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE(IntPointTest);
+    CPPUNIT_TEST(test);
+    CPPUNIT_TEST_SUITE_END();
+
+public:
+    /*!
+     * \brief Sets up the test suite to prepare for testing.
+     * 
+     * Since <em>IntPointTest</em> only has static functions, no instance
+     * needs to be created here.
+     */
+    void setUp();
```

Comment:
I don't think you need to even define this function then, right?

Reply:
I thought so.

Reply:
Ok it seems not. Still it's not a bad idea to keep it in the 'template' for unit cases.